docs/install/operator/configuring-serving-cr.md

```diff
@@ -0,0 +1,393 @@
+---
+title: "Configuring the Serving Operator Custom Resource"
+weight: 20
+type: "docs"
+aliases:
+- /docs/operator/configuring-serving-cr/
+---
+
+The Knative Serving operator can be configured with these options:
+
+- [All the ConfigMaps](#all-the-configmaps)
```

Comment:
I just read this, and it reads strangely:

"The Knative Serving operator can be configured with the option 'all the ConfigMaps'."

Maybe "Service Configuration by ConfigMap"?

You probably also want to change the last one to 'System Resource Settings', rather than saying "The Knative Serving operator can be configured with the option 'managing resources for containers'".